ci-operator/step-registry/stackrox/install/stackrox-install-commands.sh

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+OPERATOR_VERSION=${OPERATOR_VERSION:-}
+OPERATOR_CHANNEL=${OPERATOR_CHANNEL:-stable}
+
+cat <<EOF
+>>> Install ACS into a single cluster [$(date -u)].
+* Subscribe to rhacs-operator.
+* In new namespace "stackrox":
+  * Create central custom resource.
+  * Wait for Central to start.
+  * Request init-bundle from Central.
+  * Create secured-cluster custom resource with init-bundle.
+  * Wait for all services minimal running state.
+EOF
+
+echo ">>> Prepare script environment"
+export SHARED_DIR=${SHARED_DIR:-/tmp}
+export KUBECONFIG=${KUBECONFIG:-${SHARED_DIR}/kubeconfig}
+echo "SHARED_DIR=${SHARED_DIR}"
+echo "KUBECONFIG=${KUBECONFIG}"
+
+cr_url=https://raw.githubusercontent.com/stackrox/stackrox/master/operator/tests/common
+
+SCRATCH=$(mktemp -d)
+cd "${SCRATCH}"
+function exit_handler() {
+  exitcode=$?
+  set +e
+  echo ">>> End ACS install"
+  echo "[$(date -u)] SECONDS=${SECONDS}"
+  rm -rf "${SCRATCH}"
+  if [[ ${exitcode} -ne 0 ]]; then
+    echo "Failed install with ${OPERATOR_VERSION}"
+  else
+    echo "Successfully installed with ${OPERATOR_VERSION}"
+  fi
+}
+trap 'exit_handler' EXIT
+trap 'echo "$(date +%H:%M:%S)# ${BASH_COMMAND}"' DEBUG
+
+
+function get_jq() {
+  local url
+  url=https://github.com/jqlang/jq/releases/download/jq-1.7.1/jq-linux-amd64
+  echo "Downloading jq binary from ${url}"
+  curl -Ls -o ./jq "${url}"
+  chmod u+x ./jq
+  export PATH=${PATH}:${PWD}
+}
+jq --version || get_jq
+
+ROX_PASSWORD=$(oc -n stackrox get secret admin-pass -o json 2>/dev/null | jq -er '.data["password"] | @base64d') \
+  || ROX_PASSWORD="$(LC_ALL=C tr -dc _A-Z-a-z-0-9 < /dev/urandom | head -c12 || true)"
+centralAdminPasswordBase64="$(echo "${ROX_PASSWORD}" | base64)"
+cat <<EOF > central-cr.yaml
+apiVersion: platform.stackrox.io/v1alpha1
+kind: Central
+metadata:
+  name: stackrox-central-services
+spec:
+  imagePullSecrets:
+  - name: e2e-test-pull-secret
+  # Resource settings should be in sync with /deploy/common/local-dev-values.yaml
+  central:
+    adminPasswordSecret:
+      name: admin-pass
+    resources:
+      requests:
+        memory: 1Gi
+        cpu: 500m
+      limits:
+        memory: 4Gi
+        cpu: 1
+    exposure:
+      loadBalancer:
+        enabled: false
+      route:
+        enabled: false
```

Comment:
Should it be `true`? How will tests access central?
```suggestion
        enabled: true
```

Reply:
same comment as above, I would template this out.